internal/cmd/operator-sdk/run/bundle/cmd.go

```diff
@@ -30,7 +30,13 @@ func NewCmd(cfg *operator.Configuration) *cobra.Command {
 		Use:   "bundle <bundle-image>",
 		Short: "Deploy an Operator in the bundle format with OLM",
 		Long: `The single argument to this command is a bundle image, with the full registry path specified.
-If using a docker.io image, you must specify docker.io(/<namespace>)?/<bundle-image-name>:<tag>.`,
+If using a docker.io image, you must specify docker.io(/<namespace>)?/<bundle-image-name>:<tag>.
+
+The main purpose of this command is to streamline running the bundle without having to provide an index image with the bundle already included.
+
+An index image can be specified to resolve depenendencies for a bundle by specifying the ` + "`--index-image`" + ` flag.
```

Comment:
```suggestion
An index image can be specified to resolve dependencies for a bundle by specifying the ` + "`--index-image`" + ` flag.
```

Reply:
The + is so that we can "escape" the backtick ( \` ) character. This makes it so that in the documentation it shows up like `--index-image` rather than --index-image. Because the string modified in the cmd.go file uses the backtick character to make it a literal string you can't use traditional escape sequences to escape a character.

I personally think this makes the CLI docs look nicer, so I would like to keep this in.

Reply:
Ah right, this is for the help text. That makes sense. 
Thanks for clarifying @everettraven